src/services/YotiService.ts

```diff
@@ -137,9 +138,12 @@ export class YotiService {
 		countryCode: string,
 		yotiCallbackUrl: string,
 	): Promise<string | undefined> {
+		const sessionDeadlineDate = new Date(new Date().getTime() + (this.YOTI_SESSION_TTL_DAYS ? this.YOTI_SESSION_TTL_DAYS : 10) * 24 * 60 * 60 * 1000);
```

Comment:
is (`this.YOTI_SESSION_TTL_DAYS ? this.YOTI_SESSION_TTL_DAYS : 10) `required as EnvironmentVariables,ts should be setting it to default 10 days if not present?

Reply:
Is https://momentjs.com/ a good library to use to add those many days and  set the hour?

Reply:
momentjs would be good but I just didn't fancy adding another library to the codebase when it could quite simply be done outthebox but happy to use moment if you'd prefer it